cypress/integration/save.cy.js

```diff
@@ -52,6 +61,38 @@ describe('save', () => {
         deleteVisualization()
     })
 
+    it('new AO with sorted table saves correctly', () => {
+        const AO_NAME = `TEST SORTING ${new Date().toLocaleString()}`
+        setupTable()
+
+        // save with a name
+        saveVisualization(AO_NAME)
+        expectAOTitleToContain(AO_NAME)
+        expectTableToBeVisible()
+
+        // apply sorting
+        getTableHeaderCells()
+            .find(`button[title*="${TEST_DIM_INTEGER}"]`)
+            .click()
+
+        expectTableToBeUpdated()
```

Comment:
add a step that verifies that the actual values in the table are sorted? e.g. change "2, 3, 1" to "1, 2, 3".

Also add this verification step to when the AO has been loaded again below.

Reply:
There are tests checking the sorting on the values in `table.cy.js`.

But I've added a test that checks the sorting after loading the AO just saved.